cdap-docs/tools/cdap-xml-files.txt

```diff
@@ -0,0 +1,9 @@
+# Line starting with a # or space are ignored
+# Support for JSON files to be added
+
+cdap-distributions/src/etc/cdap/conf.dist/cdap-site.xml.example
+cdap-examples/Purchase/src/test/resources/cdap-site.xml
+cdap-standalone/src/main/resources/cdap-site.xml
+# cdap-ui/server/config/development/cdap.json
```

Comment:
I would like to add this file to the tests. But I haven't written the support for JSON yet, as noted on line 2.